v1/src/test/java/com/google/cloud/teleport/spanner/ExportPipelineIT.java

```diff
@@ -44,14 +46,35 @@
 import org.junit.Test;
 import org.junit.experimental.categories.Category;
 import org.junit.runner.RunWith;
-import org.junit.runners.JUnit4;
+import org.junit.runners.Parameterized;
+import org.junit.runners.Parameterized.Parameters;
 
 /** Integration test for {@link ExportPipeline Spanner to GCS Avro} template. */
 @Category(TemplateIntegrationTest.class)
 @TemplateIntegrationTest(ExportPipeline.class)
-@RunWith(JUnit4.class)
+@RunWith(Parameterized.class)
```

Comment:
Is it possible to create a `SpannerTestBase` and parameterize that so that we don't have to do this every time we right a new test? 

https://stackoverflow.com/questions/67477549/junit4-parameterized-test-extends-parameterized-base-class-need-cartesian-pr

If not , it is also possible that the test author just forgets writing the new test in this way. How will be ensure that this style of writing tests is uniformly followed in future.

Reply:
Moved the parameterization to base class. This simplifies the test structure. Thank you!